src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.cs

```diff
@@ -10,18 +10,18 @@ namespace System.Net.NetworkInformation
     public partial class NetworkChange
     {
         // The list of current address-changed subscribers.
-        private static readonly Dictionary<NetworkAddressChangedEventHandler, ExecutionContext> s_addressChangedSubscribers =
-            new Dictionary<NetworkAddressChangedEventHandler, ExecutionContext>();
+        private static readonly Dictionary<NetworkAddressChangedEventHandler, ExecutionContext?> s_addressChangedSubscribers =
+            new Dictionary<NetworkAddressChangedEventHandler, ExecutionContext?>();
 
         // The list of current availability-changed subscribers.
-        private static readonly Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext> s_availabilityChangedSubscribers =
-            new Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext>();
+        private static readonly Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext?> s_availabilityChangedSubscribers =
+            new Dictionary<NetworkAvailabilityChangedEventHandler, ExecutionContext?>();
 
         private static readonly NetworkAvailabilityEventArgs s_availableEventArgs = new NetworkAvailabilityEventArgs(isAvailable: true);
         private static readonly NetworkAvailabilityEventArgs s_notAvailableEventArgs = new NetworkAvailabilityEventArgs(isAvailable: false);
-        private static readonly ContextCallback s_runHandlerAvailable = new ContextCallback(RunAvailabilityHandlerAvailable);
-        private static readonly ContextCallback s_runHandlerNotAvailable = new ContextCallback(RunAvailabilityHandlerNotAvailable);
-        private static readonly ContextCallback s_runAddressChangedHandler = new ContextCallback(RunAddressChangedHandler);
+        private static readonly ContextCallback s_runHandlerAvailable = new ContextCallback(RunAvailabilityHandlerAvailable!);
```

Comment:
Why do these need a `!`?

Reply:
Because ContextCallback delegate accepts nullable parameter `state`, but the method `RunAvailabilityHandlerAvailable` can't have nullable `state`, here or within the method `RunAvailabilityHandlerAvailable` we will need to bang